app/views/vocab_sheets/show.html.haml

```diff
@@ -1,5 +1,3 @@
-- content_for :scripts do
```

Comment:
Is this no longer needed?

Reply:
Yep. The only thing the block was injecting into the `:scripts` section was that typo'd comment